tests/test_ciphers.py

```diff
@@ -69,7 +70,9 @@ def vectors():
             key="0123456789abcdef",
             iv="1234567890abcdef",
             plaintext=t2b("now is the time "),
-            ciphertext=h2b("959492575f4281532ccc9d4677a233cb")
+            ciphertext=h2b("959492575f4281532ccc9d4677a233cb"),
+            #ciphertext_ctr = b'(u(\xdd\xf4\x84\xb1\x05]\xeb\xbeu\x1e\xb5+\x8a'
```

Comment:
Is this comment supposed to be here?

Reply:
No, it was accidental. Thanks for spotting.